dbms/src/Functions/h3EdgeLengthM.cpp

```diff
@@ -0,0 +1,79 @@
+#include "config_functions.h"
+#if USE_H3
+#include <Columns/ColumnsNumber.h>
+#include <DataTypes/DataTypesNumber.h>
+#include <Functions/FunctionFactory.h>
+#include <Functions/IFunction.h>
+#include <Common/typeid_cast.h>
+#include <ext/range.h>
+
+
+extern "C"
+{
+#ifdef __clang__
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdocumentation"
+#endif
+
+#include <h3api.h>
+
+#ifdef __clang__
+#pragma clang diagnostic pop
+#endif
+}
+
+namespace DB
+{
+
+class FunctionH3EdgeLengthM : public IFunction
```

Comment:
Need brief docs right here.

Reply:
Should I document it in code? I wrote Russian docs about this function, this function is a wrapper around `H3` edgeLengthM function.

Reply:
Yes, we need brief documentation in code.

Reply:
Done in efdb2d2871